src/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -130,6 +130,18 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <PackageReference>
       <GeneratePathProperty>true</GeneratePathProperty>
     </PackageReference>
+
+    <CompileBefore>
+      <CompileOrder>CompileBefore</CompileOrder>
+    </CompileBefore>
+
+    <CompileAfter>
+      <CompileOrder>CompileAfter</CompileOrder>
+    </CompileAfter>
+
+    <CompileLast>
+      <CompileOrder>CompileLast</CompileOrder>
+    </CompileLast>
```

Comment:
Could you please also add `CompileFirst`?

I see its usage in `FSharpSourceCodeCompileOrder` here: https://github.com/dotnet/fsharp/blob/6cdcc3c33ca162988f3f90033f891bc265b9a3bc/src/FSharp.Build/Microsoft.FSharp.Targets#L262

and in `FSharp.Core` here: https://github.com/dotnet/fsharp/blob/6cdcc3c33ca162988f3f90033f891bc265b9a3bc/src/FSharp.Core/FSharp.Core.fsproj#L80

Reply:
@KevinRansom This hasn't been resolved. 😞